catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/TransformersCommand.java

```diff
@@ -0,0 +1,111 @@
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.catalog;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.osgi.framework.ServiceReference;
+
+import ddf.catalog.Constants;
+import ddf.catalog.transform.InputTransformer;
+import javafx.util.Pair;
+
+/**
+ * Provides information on available transformers
+ */
+@Service
+@Command(scope = CatalogCommands.NAMESPACE, name = "transformers", description = "Provides information on available transformers.")
+public class TransformersCommand extends CatalogCommands {
+
+    private static final String MIME_TYPE = "mime-type";
+    private static final String SCHEMA = "schema";
+
+    private static final String NOT_AVAILABLE = "N/A";
+    private static final String LINE = "------------------------";
+
+    HashMap<String, Pair<String, ArrayList<String>>> transformersProperties;
+    ArrayList<ServiceReference<InputTransformer>> serviceReferences;
+    
+    @Override
+    protected Object executeWithSubject() throws Exception {
+        transformersProperties = new HashMap<>();
+        serviceReferences = (ArrayList<ServiceReference<InputTransformer>>) bundleContext.getServiceReferences(InputTransformer.class, null);
+
+        for (ServiceReference ref : serviceReferences) {
+            String id = getTransformerProperty(ref, Constants.SERVICE_ID);
+
+            if (!id.equals(NOT_AVAILABLE)) {
+                String schema = getTransformerProperty(ref, SCHEMA);
+                ArrayList<String> mimeType = getTransformerMimeProperty(ref);
+
+                transformersProperties.put(id, new Pair<>(schema, mimeType));
+            }
+        }
+
+        int activeTransformers = transformersProperties.size();
+
+        if (activeTransformers == 0) {
+            console.printf("There are no active transformers%n%n");
+
+        } else {
+            console.printf("%nActive Transformers: %d%n%s%n%n", activeTransformers, LINE);
+
+            int entryCount = 1;
+            for (Map.Entry<String, Pair<String, ArrayList<String>>> t: transformersProperties.entrySet()) {
+                console.printf("%s: %s%n%n",
+                        Constants.SERVICE_ID, t.getKey());
+                console.printf("\t%s: %s%n",
+                        SCHEMA, t.getValue().getKey());
+                console.printf("\t%s: %s%n",
+                        MIME_TYPE + "s", t.getValue().getValue().remove(0));
+
+                for (String mimeType : t.getValue().getValue()) {
+                    console.printf("\t\t    %s%n", mimeType);
+                }
+
+                if (entryCount++ != activeTransformers) {
```

Comment:
✏️ You could also consider using an iterator over the collection and checking `hasNext()` here instead of maintaining a counter.